packages/slice-machine/components/ReviewModal/index.tsx

```diff
@@ -49,11 +69,65 @@ const SelectRatingComponent = ({ field, form }: FieldProps) => {
   );
 };
 
-const ReviewModal: React.FunctionComponent<ReviewModalProps> = ({
-  close,
-  isOpen,
-  onSubmit,
-}) => {
+const ReviewModal: React.FunctionComponent<ReviewModalProps> = () => {
+  const { customTypes } = useContext(CustomTypesContext);
+  const libraries = useContext(LibrariesContext);
+  const { isReviewLoading, isLoginModalOpen, hasSendAReview } = useSelector(
+    (store: SliceMachineStoreType) => ({
+      isReviewLoading: isLoading(store, LoadingKeysEnum.REVIEW),
+      isLoginModalOpen: isModalOpen(store, ModalKeysEnum.LOGIN),
+      hasSendAReview: userHasSendAReview(store),
+    })
+  );
+
+  const dispatch = useDispatch();
+  const openLogin = () =>
+    dispatch(modalOpenCreator({ modalKey: ModalKeysEnum.LOGIN }));
+  const startReviewLoading = () =>
+    dispatch(startLoadingActionCreator({ key: LoadingKeysEnum.REVIEW }));
+  const stopReviewLoading = () =>
+    dispatch(stopLoadingActionCreator({ key: LoadingKeysEnum.REVIEW }));
+  const skipReview = () => dispatch(skipReviewCreator());
+  const sendAReview = () => dispatch(sendAReviewCreator());
+
+  const { addToast } = useToasts();
+
+  const sliceCount =
```

Comment:
Nop I don't think this is a good idea, because the count here his linked to how the front manage the data and this is specific to the front side only.

Reply:
Seen with amaury : The front type shouldn't be in the Core.